server/apps/common/jwt_utils.py

```diff
@@ -30,3 +31,25 @@ def generate_jwt_refresh_token(user):
     refresh_token = jwt.encode(refresh_token_payload, settings.REFRESH_TOKEN_SECRET, algorithm='HS256').decode('utf-8')
 
     return refresh_token
+
+
+def verify_jwt_token(token):
+    if not token:
+        raise NotAuthenticated('Unauthenticated!')
+    try:
+        payload = jwt.decode(token, settings.SECRET_KEY, algorithms=['HS256'])
+    except jwt.ExpiredSignatureError:
+        raise NotAuthenticated('Authentication failed! Signature Expired!')
```

Comment:
All these strings should be from a constants file.
You can create a python file `messages.py`. And add in the strings there

Reply:
Updated